#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Default deployment is production, not development.

@DPATCH@
--- src/etc/config.ini-fhs.orig	2010-07-16 17:50:05.000000000 +0200
+++ src/etc/config.ini-fhs	2010-07-16 17:51:19.000000000 +0200
@@ -15,13 +15,13 @@ custom_path = "$core/config_path/custom"
 
 extra_config_dirs = "$core/config_path/config.ini.d"
 
-; Switch to "production" to remove extra debugging
-installation_environment = development
+; Switch to "development" or "integration" to add extra debugging
+installation_environment = production
 
 ; fine-grained control over debugging
 
 ; enable debugging at all (overlay the page with pink box)
-sysdebug_enable = true
+sysdebug_enable = false
 
 ; enable the PHP Error Handler (default if sysdebug_enable)
 sysdebug_phphandler = true
